src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -505,9 +505,9 @@ class BipDescription:
 
 
 @attr.s(frozen=True, slots=True)
-class FluidDescription:
+class CompositionalFluidDescription:
```

Comment:
Renaming classes is an api breakage, alfasim-sdk is on version 0 but I don't think we should get used to this.

Reply:
I see, honestly we should try to be more specific during naming so that we don't let something generic like this happen so often, but I guess the only other option now (assuming we couldn't rename) would be to make FluidDescription contain both compositional and combined fluids

Reply:
@nicoddemus 
Are you in favor we make that breakage now?
No many ppl are using it and we are in version ´0`.

Reply:
Yeah I think this is fine for now, we just need to warn the PWPa folks.

Reply:
Make sure to update the https://github.com/ESSS/alfasim-sdk/blob/master/CHANGELOG.rst too, including all breaking changes.